service/src/main/resources/application.yml

```diff
@@ -30,6 +30,7 @@ env:
   urls: # While we've traditionally thought of these as env specific and ok to hardcode, with kubernetes they may change
     sam: ${SAM_ADDRESS:https://sam.dsde-dev.broadinstitute.org/}
     terra-datarepo: ${TERRA_DATAREPO_URL:https://jade.datarepo-dev.broadinstitute.org}
+    bpm: ${BPM_ADDRESS:https://bpm.dsde-dev.broadinstitute.org/}
```

Comment:
cc @aherbst-broad 
WSM now knows the BPM URL, currently only used by the LZ amalgamation.
Helm support to add it per environment is here: https://github.com/broadinstitute/terra-helmfile/pull/3374/